plugins/wasi_nn/wasinnenv.cpp

```diff
@@ -4,6 +4,8 @@
 #include "wasinnenv.h"
 #include "wasinnmodule.h"
 
+#include <sstream>
```

Comment:
Is this for debugging?

Reply:
Nope, this is for fixing https://github.com/WasmEdge/WasmEdge/issues/2811